sheepdog_exporter/sheepdog_exporter.py

```diff
@@ -391,6 +391,10 @@ def is_type_data_category(self, program, project, my_type):
         '''
         data_file_categories = ['data_file', 'metadata_file', 'index_file']
         type_schema = self.get_schema_for_type(program, project, my_type)
+        for property_key in type_schema['properties']:
+            description = type_schema['properties'][property_key].get('description', 'blank')
+            if 'HARMONIZED' in description:
+                print('type:', my_type, 'prop', property_key, description)
```

Comment:
Although I (almost) always prefer logging over `print`, in this case `print` is fine wrt the output and consistent with the rest of the program. I suggesting continuing to use `print` in this case (as is currently done) rather than switch to logging.